xmtp_mls/src/groups/mls_sync.rs

```diff
@@ -199,8 +201,18 @@ where
             self.client.inbox_id(),
             epoch
         );
-        self.maybe_update_installations(&mls_provider, None).await?;
 
+        // Also sync the "stitched DMs", if any...
+        for other_dm in conn.other_dms(&self.group_id)? {
+            let other_dm =
+                Self::new_from_arc(self.client.clone(), other_dm.id, other_dm.created_at_ns);
```

Comment:
I don't think this should cause an issue with @mchenani 's global MLS group lock.